VideoLocker/res/values/styles.xml

```diff
@@ -374,24 +371,19 @@
         <item name="android:layout_marginTop">@dimen/spinnerPopupBorder</item>
         <item name="android:paddingBottom">@dimen/spinnerPopupBorder</item>
         <item name="android:overlapAnchor" tools:targetApi="21">true</item>
```

Comment:
While the spinner dropdown is overlapping by default in the material theme, that is an implementation detail, and may be subject to change in later themes (just as it was before). Since this is a design requirement, it should be explicitly defined in the style.